tee-worker/cli/demo_sidechain.sh

```diff
@@ -110,7 +110,9 @@ echo ""
 ${CLIENTWORKER1} trusted --mrenclave ${MRENCLAVE} --direct set-balance ${ICGACCOUNTBOB} 0
 
 echo "Get balance of Alice's incognito account (on worker 1)"
-${CLIENTWORKER1} trusted --mrenclave ${MRENCLAVE} balance ${ICGACCOUNTALICE}
+# ${CLIENTWORKER1} trusted --mrenclave ${MRENCLAVE} balance ${ICGACCOUNTALICE}
+# ICGACCOUNTALICE's public key is 0x50503350955afe8a107d6f115dc253eb5d75a3fe37a90b373db26cc12e3c6661
+${CLIENTWORKER1} trusted --mrenclave ${MRENCLAVE} get-storage System Account 0x50503350955afe8a107d6f115dc253eb5d75a3fe37a90b373db26cc12e3c6661
```

Comment:
Is it possible to replace all the getter commands (`balance`, `nonce`) with `get-storage`? Just comment out the original command for reference

Reply:
Done